rpc/send.go

```diff
@@ -86,6 +86,29 @@ func (s SendError) Error() string {
 // CanRetry implements the Retryable interface.
 func (s SendError) CanRetry() bool { return s.canRetry }
 
+//StoreAddr is used to identify the address of replication
+//One node may have multiple stores, so StoreID is needed
+type StoreAddr struct {
+	Addr    net.Addr
+	StoreID proto.StoreID
+}
+
+func (addr *StoreAddr) String() string {
+	return addr.Addr.String() + fmt.Sprint(addr.StoreID)
+}
+
+type storeClient struct {
+	*Client
+	storeID proto.StoreID
+}
+
+func (client *storeClient) getStoreAddr() *StoreAddr {
```

Comment:
This function does not need to return a pointer. Simpler if it just returns `StoreAddr` and less memory allocation too.